### ISSUE TITLE
Installing glice fails

### ISSUE THREAD

**Issue by weeco** [ribice/glice#29]
I'm having issues installing glice, partially because v2.0.0 for instance has been released but go.mod does not comply with what Go expects (`/v2` has to be the suffix), but also all the README commands do not seem to work. I have installed other CLI tools without issues. Here's a screenshot:

![CleanShot 2022-01-13 at 21 12 15](https://user-images.githubusercontent.com/23424570/149402426-8e4468e0-5e46-49ce-8d72-1b22076cf583.png)

As for the v2 go.mod name, see here how major versioning in Go works (not very intuitive :D): https://go.dev/blog/v2-go-modules

```
go install github.com/ribice/glice/cmd@v2.0.0
go install: github.com/ribice/glice/cmd@v2.0.0: github.com/ribice/glice@v2.0.0: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```

**Comment by derekbassett** [comment]
Bump?

**Comment by ribice** [comment]
Should be resolved with f1a5060108c86c545699d8842eac78d953dd7121.

Do `go install github.com/ribice/glice/v2/cmd/glice@v2.3.0`.

If the issue persists please re-open the issue. Thanks!
